selfdrive/car/hyundai/values.py

```diff
@@ -314,8 +314,10 @@ class CAR(Platforms):
     flags=HyundaiFlags.EV,
   )
   HYUNDAI_IONIQ_6 = HyundaiCanFDPlatformConfig(
-    [HyundaiCarDocs("Hyundai Ioniq 6 (with HDA II) 2023", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_p]))],
-    HYUNDAI_IONIQ_5.specs,
+    [HyundaiCarDocs("Hyundai Ioniq 6 (with HDA II) 2023-24", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_p]))],
+    # Average of min/max mass of SEL/LTD models. Significantly lighter SE Standard Range should be uncommon
+    # https://www.hyundainews.com/assets/documents/original/53031-2023IONIQ6ProductSpecs093022.pdf
+    CarSpecs(mass=2004, wheelbase=2.95, steerRatio=14.27, tireStiffnessFactor=0.65),
```

Comment:
Expand up to see old values in IONIQ 5. Not sure if this is the right way to pick a weight, or what effect it has. The IONIQ 5 is heavier, but it looks like a value on the lower side was picked.